rust/connlib/tunnel/src/tests.rs

```diff
@@ -135,11 +147,21 @@ impl StateMachineTest for TunnelTest {
             allocations: ref_state.relay.allocations.clone(),
             buffer: ref_state.relay.buffer.clone(),
         };
+
         client.state.update_relays(
             HashSet::default(),
             HashSet::from([relay.explode("client")]),
             ref_state.now,
         );
+        let _ = client.state.update_interface_config(Interface {
```

Comment:
Is the `let _ = ` here for discarding an unused `Result`, even if it's an error?

Apparently the `let` is optional now https://stackoverflow.com/a/51142002

I usually do `.ok()` but I'll switch to this style to match existing code

Reply:
It is because it returns a `bool` and is tagged with `must_use` because we need to trigger a side-effect here in the production code.